### DISCUSSION TITLE
Whisper output different on different gpus & why open ai whisper cloud is better than what i am locally running?

### DISCUSSION THREAD

**Discussion by Oheed911** [openai/whisper#2209] [Q&A]
Hi, i wanted to ask about my recent problem with whisper. 

I am running whisper on two different gpus, one is A4000 and the other is A5000. I have same audio file and same code. it is producing very different audio, what could be the possible issue with this?

More over, there is a significant difference between the results of opena ai whisper cloud and my local open ai whisper? 

I am using large-v2 model. 

do let me know about this. your help is appreciated. Thanks!

**Comment by phineas-pta** [comment]
see #81
